### ISSUE TITLE
pvr seekbar - channel logo in line with seekbar

### ISSUE THREAD

**Issue by loggio** [jurialmunkey/skin.arctic.horizon#259]
As the title suggests, seems rather odd having the channel logo floating in the top left corner when cycling channels in full screen. 
can we have this moved in line with the channel title as part of the seekbar?

![screenshot104](https://user-images.githubusercontent.com/16231443/91378686-62316780-e864-11ea-869c-5ed05b948038.png)

**Comment by jurialmunkey** [comment]
The problem is that there's a huge variation in pvr channel logos which makes any positioning really difficult. In some setups logos will be relatively squarish whereas other will be a wide and short rectangle.

Even worse is that often there will be instances where only *some* channels have logos which means either the info is moving all over the place or there is a massive gap on the side which looks off.

Putting the channel logo in the top corner was my solution to all these variations.

**Comment by loggio** [comment]
Fair enough. It's just that nearly every skin has always done it this way. I thought a simple fixed width and height  and to keep the aspect ratio was the normal. Then just a visibly condition for the title position if there is no logo present.

Anyways, no real issue I can always hack this in if it gets the better of me.
